app/views/api/v1/accounts/contacts/contact_inboxes/create.json.jbuilder

```diff
@@ -1 +1 @@
-json.partial! 'api/v1/models/contact_inbox', formats: [:json], resource: @contact_inbox
+json.partial! 'api/v1/models/contact_inbox', formats: [:json], resource: @contact_inbox, hide_channel_credentials: true
```

Comment:
we won't need this here right ? we only need to add this where we including the inbox partial